src/fuzz_introspector/analyses/calltree_analysis.py

```diff
@@ -170,17 +202,28 @@ def html_create_dedicated_calltree_file(
         complete_html_string += html_header
 
         # Display fuzz blocker at top of page
-        fuzz_blockers = self.get_fuzz_blockers(
-            profile,
-            max_blockers_to_extract=12
-        )
+        if profile.branch_blockers:
+            blockers_node_map = self.collect_calltree_nodes(profile.branch_blockers[:12],
+                                                            profile.function_call_depths)
+            fuzz_blocker_nodes = list(blockers_node_map.values())
+            fuzz_blocker_table = self.create_branch_blocker_table(
+                profile,
+                [],
+                "",
+                12
+            )
+        else:
```

Comment:
I think it would be nice to, down the line, to provide the option for selecting what type(s) of blocker to include

Reply:
Yes, that makes sense.